apps/server/test/lexical/server/code_intelligence/completion/env_test.exs

```diff
@@ -739,4 +864,62 @@ defmodule Lexical.Server.CodeIntelligence.Completion.EnvTest do
                sort_items(items)
     end
   end
+
+  describe "prefix_aliases/1" do
+    test "when in current module" do
+      source = ~q<
+        defmodule User do
+          defstruct [:name, :age]
+
+          def name(%__MODULE__{|}) do
+          end
+        end
+      >
+
+      assert [:__MODULE__] == source |> new_env() |> prefix_aliases()
+    end
+
+    test "when in a current module child" do
+      source = ~q<
+        defmodule User do
+          defmodule Profile do
+            defstruct [:info]
+          end
+
+          def name(%__MODULE__.Profile{|}) do
+          end
+        end
+      >
+
+      assert [:__MODULE__, :Profile] == source |> new_env() |> prefix_aliases()
+    end
+
+    test "when in a struct" do
+      source = ~q<
+        defmodule User do
+          defstruct [:name, :age]
+
+          def name(%User{|}) do
+          end
+        end
+      >
+
+      assert [:User] == source |> new_env() |> prefix_aliases()
+    end
+
+    test "when in a struct child" do
+      source = ~q<
+        defmodule User do
+          defmodule Profile do
+            defstruct [:info]
+          end
+
+          def name(%User.Profile{|}) do
+          end
+        end
+      >
+
+      assert [:User, :Profile] == source |> new_env() |> prefix_aliases()
```

Comment:
So does prefix aliases just get the referenced modules in the `env`'s prefix? I think there is a better name than that, like `module_references` . 

Though I'm still struggling with `prefix_aliases`, I don't know what it does really.
What would it return for 
```elixir
foo = %Foo.User{name: "Steve", account: %Account{}}|
```
`[[:Foo, :User], :Account]`, or [:Foo, :User, :Account]`?

Reply:
it should return []